server/controllers/board.js

```diff
@@ -1,97 +1,149 @@
 const asyncHandler = require("express-async-handler");
 const Board = require("../models/Board");
-const { v4: uuidv4, v4 } = require("uuid");
+const User = require("../models/User");
 
-exports.getBoards = asyncHandler(async (req, res) => {
-  const boards = await Board.find();
+exports.getBoard = asyncHandler(async (req, res) => {
+  const board = await Board.findOne({ _id: req.params.id }).populate({
+    path: "columns",
+    populate: {
+      path: "cards",
+      model: "card",
+    },
+  });
 
-  return res.json(boards);
+  if (!board) {
+    res.status(404);
+    throw new Error("Board not found");
+  }
+
+  return res.status(200).json(board);
 });
 
-exports.getBoard = asyncHandler(async (req, res) => {
-  try {
-    const board = await Board.findById(req.params.id);
+exports.createBoard = asyncHandler(async (req, res) => {
+  const { name } = req.body;
+
+  const newBoard = await Board.create({
+    name,
+    user: req.params.id,
+  });
 
-    if (!board) res.status(404).json({ error: "Board not found" });
+  console.log(newBoard.name);
+  // For testing with postman
+  // const newBoard = await Board.create({
+  //   name,
+  //   user: "60b91df75edef24420936968",
+  // });
```

Comment:
We don't need to push commented code into the codebase. 👍🏻

Reply:
It's gone!